### PR TITLE
Http sns doc update

### PR DESCRIPTION
**Pull request by alindfor**
Proposed changes include:
Add information about keys in url-encoded form for HTTP notifications
Add information about JSON schema for Amazon SNS notifications

### CODE REVIEW

File: doc/modules/mod_event_pusher_http.md [hunk 1]
```diff
@@ -44,6 +44,20 @@ This module uses a connection pool created by mongoose_http_client. It must be d
 
 Notifications will be POSTed to `http://localhost:8000/webservice/notifications`.
 
+## Payload format
+The HTTP event pusher sends a POST request with Content-Type `application/x-www-form-urlencoded`. The form has the following fields:
+* `author`: lower case username of the user who authored the message
```
Comment (fenek) [inline doc/modules/mod_event_pusher_http.md]: These fields are not only converted to lower case but actually are processed by specific profiles of "stringprep", as per https://tools.ietf.org/html/rfc6122. You may mention lower case conversion as one of the consequences of this process of course.

File: doc/modules/mod_event_pusher_http.md [hunk 2]
```diff
@@ -44,6 +44,20 @@ This module uses a connection pool created by mongoose_http_client. It must be d
...
+* `server`: . lower case name of the server from where the message originates
+* `receiver`: lower case username of the user who the message is for
+* `message`: message text
```
Comment (fenek) [inline doc/modules/mod_event_pusher_http.md]: "Payload from `<body/>` element of the message." would be more precise.

File: doc/modules/mod_event_pusher_sns.md [hunk 3]
```diff
@@ -56,4 +56,42 @@ Full topics for notifications (ARN as defined in [Amazon Resource Names][aws-arn
...
+#### Sent messages
+The JSON format for a private message notification is:
+```JSON
+{
+    "to_user_id": "bob0.61585@localhost",
+    "message": "Hello, Bob",
+    "from_user_id": "alice0.61585@localhost"
+}
+```
+The notification is similar for group messages except that the `to_user_id` is the recipient room jid. For example:
```
Comment (fenek) [inline doc/modules/mod_event_pusher_sns.md]: Room **JID** (not jid :)).

File: doc/modules/mod_event_pusher_sns.md [hunk 4]
```diff
@@ -56,4 +56,42 @@ Full topics for notifications (ARN as defined in [Amazon Resource Names][aws-arn
...
+
+The JSON format for an online presence update notification is:
+```JSON
+{
+    "user_id": "alice0.61585@localhost",
+    "present": true
+}
+```
+
+For offline presence updates the `present` boolean value is set to false:
```
Comment (goddammit) [inline doc/modules/mod_event_pusher_sns.md]: For offline presence updates, the `present` boolean value is set to false ?

**Pull request merged**